modules/slack/blocks/block_kit_examples/new_join_immediate.json

```diff
@@ -0,0 +1,93 @@
+{
+	"blocks": [
+		{
+			"type": "section",
+			"block_id": "immediate_welcome_main_text",
+			"text": {
+				"type": "mrkdwn",
+				"text": "Hello <google.com|@JulioMendez>! Welcome to Operation Code! I'm a bot designed to help you navigate this Slack workspace. Our goal here at Operation Code is to get veterans and their families started on the path to a career in tech. We do that through providing you with scholarships, mentoring, career development opportunities, conference tickets, and more! You can check out more information about us <https://operationcode.org/about|here>."
+			}
+		},
+		{
+			"type": "section",
+			"text": {
+				"type": "mrkdwn",
+				"text": "Much of the provided aid requires veteran or military spouse status. Please verify your status on your profile at https://operationcode.org if you haven't already."
```

Comment:
Does the veteran/milspouse status from the profile get used programmatically anywhere? I know we're working on our data collection strategy and this probably goes along with that, just curious if you saw while you were in here

Reply:
I don't believe so, no - the data is there though so we can use it down the road if we want.